### PR TITLE
CNEDT_227_223

### PR DESCRIPTION
**Pull request by mshah001**
Created 1st Karate test scenario and installed karate in docker

### CODE REVIEW

**Review by sveselev — APPROVED** [review]

File: services/test/episync_Dockerfile
Comment (darrenenq) [inline services/test/episync_Dockerfile:1]: This file needs to be named "Dockerfile"

**Review by darrenenq — COMMENTED** [review]
Let's rename the directory "question_bank" to "episync" within that we can have a variety of tests, not just question bank tests

**Review by darrenenq — COMMENTED** [review]
For naming branches and PR's use the precise name of the single parent ticket such as CNEDT-227 so Jira can understand it.

File: services/test/episync_Dockerfile
Comment (mshah001) [reply services/test/episync_Dockerfile:1]: updated and renamed

**Pull request closed without merging**